### DISCUSSION TITLE
https://steinkamp.us/posts/2024-10-16_community-comet-support-group

### DISCUSSION THREAD

**Discussion by giscus** [zsteinkamp/steinkamp.us#20] [Announcements]
# https://steinkamp.us/posts/2024-10-16_community-comet-support-group

I had a wonderful night meeting some lovely people in my community, with a comet backdrop.

https://steinkamp.us/posts/2024-10-16_community-comet-support-group

**Comment by sanathkumarbs** [comment]
Love it and such amazing pictures! We saw it hovering over Sierras and Yosemite. Such a majestic one! Not great quality as we didn't have our DSLR and were coming back from a hike!

https://imgur.com/OS0eij4

**Reply by zsteinkamp** [reply]
Oh what a great photo, Sanath! I'm sure in the dark skies where you were it's was a lot more visible than it was here. Thanks for sharing :) Let's catch up in person soon. :)
